source/Find_Flush.py

```diff
@@ -0,0 +1,64 @@
+from CardGames import *
+from River_Class import *
+
+def find_flush(player, river):
+    ### Need to have separate variables to keep track of how many cards are of each suit"
+    num_spades = 0
+    num_clubs = 0
+    num_hearts = 0
+    num_diamonds = 0
+    ### Need to have separate variables to keep track of the highest suit card
+    high_spade = Card("test", 0, [], [])
+    high_club = Card("test", 0, [], [])
+    high_heart = Card("test", 0, [], [])
+    high_diamond = Card("test", 0, [], [])
+    ### Add both Player's hand and the river
+    Cards= player.hand + river.hand
+    ### For every card, check to find the suit, then add 1 to the count
+    ### for that specific suit and see if that's the highest card in the suit
```

Comment:
I think you chose the best way to find the flush

Reply:
Thanks